export function rethrow(err: Error): void {
    throw err;
}
